### ISSUE TITLE
you ever thought about building csfml with zig

### ISSUE THREAD

**Issue by IbrahimOuhamou** [Guigui220D/zig-sfml-wrapper#59]
in the [raylib wrapper](https://github.com/Not-Nik/raylib-zig) it just uses the package in [the official ratlib repo](https://github.com/raysan5/raylib) and makes a wrapper arround it

**Comment by Guigui220D** [comment]
Yes id love to do that eventually, but i dont really have the time or energy lately
If you wanna try to tackle that issue, go ahead!

**Comment by IbrahimOuhamou** [comment]
I have never worked with cmake
